tox.ini

```
@@ -1,9 +1,9 @@
 [tox]
 envlist =
     flake8
-    py{38,39,310}-dj{32,41}-wt{41,42,50,51}
-    py311-dj41-wt{41,42,50,51}
-    py311-dj42-wt{50,51}
+    py{38,39,310}-dj{32,41}-wt{41,42,50,51,52}
+    py311-dj41-wt{41,42,50,51,52}
+    py311-dj42-wt{50,51,52}
```

Since this is a Wagtail repo, might be good to drop Wagtail 4.2 and 5.0.

```suggestion
    py{38,39,310}-dj{32,41}-wt{41,51,52}
    py311-dj41-wt{41,51,52}
    py311-dj42-wt{51,52}
```